tomviz/python/ManualManipulation.py

```diff
@@ -0,0 +1,84 @@
+def apply_shift(array, shift):
+    if shift is None:
+        return
+
+    import numpy as np
+
+    array[:] = np.roll(array, shift[0], axis=0)
+    array[:] = np.roll(array, shift[1], axis=1)
+    array[:] = np.roll(array, shift[2], axis=2)
+
+
+def apply_rotation(array, rotation, spacing):
+    import numpy as np
+
+    if rotation is None or all(np.isclose(x, 0) for x in rotation):
+        # No rotation. Nothing to do.
+        return
+
+    if not array.dtype == np.float32:
+        raise Exception('Input array must be float32')
+
+    # Wait to import ITK until we know we have to
+    import itk
```

Comment:
Will our packaged version of ITK do here?

Reply:
Yes it should! In fact, I just tested it in tomviz 1.9.0 and it worked.